src/logic/calculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
```

Comment:
Your app is not functioning as expected. The `a/c` button is not responding to click. Kindly fix that.

![image](https://user-images.githubusercontent.com/10332499/164569453-2e79816d-26ec-4169-8cd9-db37c77ccab0.png)